mmv1/products/workbench/Instance.yaml

```diff
@@ -176,6 +185,9 @@ properties:
         name: vmImage
         custom_flatten: templates/terraform/custom_flatten/workbench_instance_vm_image_flatten.go.erb
         default_from_api: true
+        exactly_one_of:
```

Comment:
For nested fields this needs to be the full terraform path i.e. `gce_setup.0.field`. I don't think we want to use exactly_one_of though, since that will make the field(s) required and that's backwards incompatible. Instead, we want `conflicts` which will stop them being specified at the same time.